unity-ggjj/Assets/Scripts/Evidence/EvidenceMenuItem.cs

```diff
@@ -10,7 +11,38 @@ public class EvidenceMenuItem : MonoBehaviour, ISelectHandler
 
     [SerializeField, Tooltip("The evidence menu this evidence menu item is a child of")]
     private EvidenceMenu _evidenceMenu;
+    
     private ICourtRecordObject _courtRecordObject;
+    private Selectable _selectable;
+
+    private Selectable Selectable
+    {
+        get
+        {
+            if (_selectable == null)
+            {
+                _selectable = GetComponent<Selectable>();
+            }
+
+            return _selectable;
```

Comment:
```suggestion
            _selectable == null ? _selectable = GetComponent<Selectable>() : _selectable;
```
Would this work as a ternary?